Assets/Scripts/UI/InGame/HUD/HUD.cs

```diff
@@ -1,41 +1,67 @@
 ﻿using System;
 using UnityEngine;
 using UnityEngine.UI;
+using TMPro;
 
 namespace Assets.Scripts.UI.InGame.HUD
 {
     public class HUD : UiElement
     {
         public GameObject Damage;
         public Labels Labels;
+        public GameObject BloodSmear;
+        public UnityEngine.Sprite[] BloodSmearSprite;
+        public bool inEditMode;
+        public bool isActive = true;
         public Crosshair Crosshair;
+        
+        public void Update()
+        {
+            if(UnityEngine.Input.GetKeyDown(KeyCode.J))
+            {
+                SetDamage(UnityEngine.Random.Range(1000,5000));
+            }
+        }
 
         public void SetDamage(int damage)
         {
-            var damageLabels = Damage.GetComponentsInChildren<Text>();
+            var damageLabels = Damage.GetComponentsInChildren<TMP_Text>();
             foreach (var label in damageLabels)
             {
                 label.fontSize = ScaleDamageText(damage);
                 label.text = damage.ToString();
             }
 
             ShowDamage();
+            ShowBloodSmear(damage);
         }
 
+
         public void ClearDamage()
         {
-            var damageLabels = Damage.GetComponentsInChildren<Text>();
+            var damageLabels = Damage.GetComponentsInChildren<TMP_Text>();
             foreach (var label in damageLabels)
             {
-                label.text = string.Empty;
+                label.text = "";
             }
+            Damage.transform.localScale = new Vector3(0,0,1f);
         }
 
         private void ShowDamage()
         {
             Damage.GetComponent<Animator>().SetTrigger("ShowDamage");
         }
 
+        private void ShowBloodSmear(int damage)
+        {
+            if(damage >= 1000)
```

Comment:
Why would a BloodSmear effect only appear above 1k damage? With +1k damage you already have the "reward" of a different color damage number. Rather make this 500